libs/btp/navigation/navigation-component.token.ts

```diff
@@ -1,11 +1,12 @@
 import { computed, Signal, TemplateRef } from '@angular/core';
 import { Nullable } from '@fundamental-ngx/cdk/utils';
 import { FdbNavigationListItemComponent } from './navigation-list-item-component.token';
+import { FdbNavigationMode, FdbNavigationState, FdbNavigationType } from './navigation.types';
 
 export abstract class FdbNavigationComponent {
-    abstract state: Signal<'expanded' | 'snapped' | 'popup'>;
-    abstract type: Signal<'horizontal' | 'vertical'>;
-    abstract mode: Signal<'desktop' | 'tablet' | 'phone'>;
+    abstract state: Signal<FdbNavigationState>;
+    abstract type: Signal<FdbNavigationType>;
+    abstract mode: Signal<FdbNavigationMode>;
```

Comment:
Can you use the mode from the shared types? Me and Inna used that together and would be nice to have it here as well